build.gradle

```diff
@@ -17,6 +17,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
         google()
         mavenCentral()
+        maven { url 'https://developer.huawei.com/repo/' }
```

Comment:
Can we please limit this to only the artifacts that we actually need to pull from this repo?

Reply:
Ok, In need to add it to two modules, the one that implement Huawei pushes and our sample app

Reply:
It is needed to apply the huawei plugin :(